modules/centrality-algorithms/index.adoc

```diff
@@ -0,0 +1,22 @@
+= Centrality Algorithms
+:description: Overview of centrality algorithms.
+
+Centrality algorithms are one way to determine the importance of a vertex to a graph, based on the importance of a vertex to other vertices.
+These algorithms assign a score to a vertex or edge based on some concept of being close to the center of connection-based activity.
+
+GSQL includes the following centrality algorithms:
```

Comment:
The algorithms are written in GSQL, but it's not accurate to say that "GSQL includes the algorithms".

The TigerGraph Graph Data Science Library provides the following centrality algorithms: